### PR TITLE
parse bpf helper funcs

### PR DESCRIPTION
**Pull request by roikol**
add libbpfgo helper to parse bpf helpers ID to string

### CODE REVIEW

File: helpers/argumentParsers.go [hunk 1]
```diff
@@ -1833,6 +1833,673 @@ func ParseBPFProgType(rawValue uint64) (BPFProgType, error) {
 	return v, nil
 }
 
+// BPFHelpers is an enum as defined in https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h
```
Comment (geyslan) [inline helpers/argumentParsers.go:1836]: This statement is a bit ambiguous.

Perhaps,

`// BPFHelpers is an enumeration of BPF helpers defined in https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h`

WDYT?

File: helpers/argumentParsers.go [hunk 2]
```diff
@@ -1833,6 +1833,673 @@ func ParseBPFProgType(rawValue uint64) (BPFProgType, error) {
...
+type BPFHelpers uint32
```
Comment (geyslan) [inline helpers/argumentParsers.go:1837]:
```suggestion
type BPFHelper uint32
```

File: helpers/argumentParsers.go (hunk 1 above)
Comment (roikol) [reply helpers/argumentParsers.go:1836]: sure!

i copied it from the `BPFProgType`: https://github.com/aquasecurity/libbpfgo/blob/main/helpers/argumentParsers.go#L1709

i'll change both of them

**Comment by roikol** [comment]
thanks for the review @geyslan !
i've updated the PR

**Review by geyslan — APPROVED** [review]
LGTM

**Review by geyslan — APPROVED** [review]

**Pull request merged**